### PR TITLE
Covfefe Particle

### PR DESCRIPTION
**Pull request by carlosrpg**

### CODE REVIEW

File: game/scripts/vscripts/items/trumps_fists.lua [hunk 1]
```diff
@@ -21,13 +21,29 @@ function modifier_item_trumps_fists_passive:IsPurgable()
...
   self.heal_prevent_duration = self:GetAbility():GetSpecialValueFor( "heal_prevent_duration" )
+
+  if IsServer() then
+    local unit = self:GetCaster()
+
+    if unit.hero_projectile == nil then
+      unit.hero_projectile = unit:GetRangedProjectileName()
+    end
+
+    self.item_projectile = "particles/items/trumps_fists/trumps_fists_projectile.vpcf"
```
Comment (Trildar) [inline game/scripts/vscripts/items/trumps_fists.lua]: There's no reason to save this on self if you aren't going to reference it anywhere else.

File: game/scripts/vscripts/items/trumps_fists.lua [hunk 2]
```diff
@@ -21,13 +21,29 @@ function modifier_item_trumps_fists_passive:IsPurgable()
...
+    unit:SetRangedProjectileName(self.item_projectile)
+  end
+end
+
+function modifier_item_trumps_fists_passive:OnDestroy()
+  if IsServer() then
+    self:GetCaster():SetRangedProjectileName( self:GetCaster().hero_projectile )
```
Comment (Trildar) [inline game/scripts/vscripts/items/trumps_fists.lua]: Once you have to call `GetX` more than once, it's generally a good idea to just make a variable for it.

**Comment by Trildar** [comment]
If only `MODIFIER_PROPERTY_PROJECTILE_NAME` actually worked.

File: game/scripts/vscripts/items/trumps_fists.lua [hunk 3]
```diff
@@ -21,13 +21,29 @@ function modifier_item_trumps_fists_passive:IsPurgable()
...
-
   self.heal_prevent_duration = self:GetAbility():GetSpecialValueFor( "heal_prevent_duration" )
+
+  if IsServer() then
+    local unit = self:GetCaster()
+
+    if unit.hero_projectile == nil then
+      unit.hero_projectile = unit:GetRangedProjectileName()
+    end
+
```
Comment (chrisinajar) [inline game/scripts/vscripts/items/trumps_fists.lua]: using `.editorconfig` should prevent whitespace lines like this. this is what's causing the tests to fail...

Comment (carlosrpg) [reply game/scripts/vscripts/items/trumps_fists.lua]: I did the change by browser on Github, my bad

**Pull request merged**